database/user_test.go

```diff
@@ -65,7 +62,11 @@ func TestEmail_Validate(t *testing.T) {
 func TestUser_saltAndPepper(t *testing.T) {
 	initEnv()
 	salt := []byte("this is a salt")
-	pepper := []byte("this is some pepper")
+	pepper, ok := os.LookupEnv(envPepper)
+	if !ok {
+		os.Setenv(envPepper, "this is a salt")
```

Comment:
Why set this env var here? Also, isn't `pepper` empty?

Reply:
The idea is that the environment variable might or might not be set.

If it's set - I want to use it as it is.

If it's not set I need to set it (because `saltAndPepper` depends on it) and then unset it again, in order to clean up the environment.

In theory, the pepper env var should not be empty because of the call to `initEnv` which sets it.

Reply:
I might be more cautious than I should be and I can just set the env var if it's not set and be done with it. If you think that would be god enough, I'll just make that change.

Reply:
I think I would just set the expected `pepper`, we don't need to set/unset the env var because we set it in `initEnv`. We also do an env var lookup for pepper in `saltAndPepper()`, so there is no point in doing it here. I would refactor out the pepper from `initEnv` and use it as the input pepper on line 78. Currently the `pepper` on line 78 would be empty if this lookup check failed